app/controllers/settings/identity_proofs_controller.rb

```diff
@@ -36,10 +42,22 @@ def create
   private
 
   def check_required_params
-    redirect_to settings_identity_proofs_path unless [:provider, :provider_username, :token].all? { |k| params[k].present? }
+    redirect_to settings_identity_proofs_path unless [:provider, :provider_username, :username, :token].all? { |k| params[k].present? }
   end
 
   def resource_params
     params.require(:account_identity_proof).permit(:provider, :provider_username, :token)
   end
+
+  def publicize_new_proof
```

Comment:
This method should have a `?` at the end for readability. Could also just name it `publish_proof?` for simplicity